firestore/testapp/README.md

```diff
@@ -13,7 +13,19 @@ The testapp performs the following:
 -   Gets a pointer to firebase::Auth, and signs in anonymously. This allows the
     testapp to access a Firebase Firestore instance with authentication rules
     enabled.
--   TODO(varconst): describe the Firestore-specific logic
+-   Initializes a Firestore instance, and sets its logging level to
+    `kLogLevelDebug` in order to see debug messages in the logs.
+-   Tests that it can create Timestamp, SnapshotMetadata, and GeoPoint objects.
```

Comment:
Nit: can you wrap `Timestamp`, etc. in backticks for consistency.

Reply:
Done.